modules/manage/partials/mountable-topics.adoc

```diff
@@ -1,8 +1,8 @@
-For topics with Tiered Storage enabled, you can mount and unmount topics to transfer the topic data between your cluster and object storage. This allows you to free up and reclaim unused partition space, or migrate a topic to a different cluster and hibernate or decommission the topic or even the entire cluster.
+For topics with Tiered Storage enabled, you can unmount a topic to detach segment data that is still on disk to object storage, and unmount a topic from object storage to attach the topic data to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up and reclaim system resources taken up by the topic, or migrate a topic to a different cluster.
```

Comment:
Seems odd that it is called Mountable, but most of the content here is about unmounting.

Reply:
free up/reclaim system resources are synonymous. Keep one or the other, but not both

Reply:
@Feediver1 That's my mistake, I'll change it.

Reply:
@Deflaimun (@mattschumpert to correct me if this is wrong) It's detaching the topic. So most of topic data would have already been uploaded to Tiered Storage (based on how quickly users have configured segment data to be moved to the cloud). Unmounting takes what is still on segment in the disk, moves it to the cloud (and stops reads and writes) so that it's all ready to "attach" again to a cluster.